src/lang-promql/complete/hybrid.ts

```diff
@@ -43,7 +43,7 @@ import { walkBackward, walkThrough } from '../parser/path-finder';
 import { aggregateOpModifierTerms, aggregateOpTerms, binOpModifierTerms, binOpTerms, functionIdentifierTerms, matchOpTerms } from './promql.terms';
 import { Completion, CompletionContext, CompletionResult, snippet } from '@codemirror/next/autocomplete';
 
-interface AutoCompleteNode {
+export interface AutoCompleteNode {
```

Comment:
I don't see the point of exporting this term. I would prefer that the file promql.term.ts doesn't depend to anything in order to avoid a nasty circular dependency

Reply:
It's because of the code that modifies the terms array to insert the "detail" after the array is created (so TypeScript requires the type)

Anyway, it's no longer necessary as I will put the detail directly like your suggestion :+1:

Reply:
Removed in https://github.com/prometheus-community/codemirror-promql/pull/55#discussion_r498716990